### PR TITLE
Add documentation for remote hosts upstream

### PR DESCRIPTION
**Pull request by mkosiarc**
Documents what remote hosts are and how to configure them

CLOUDBLD-9877

Signed-off-by: mkosiarc <mkosiarc@redhat.com>

### CODE REVIEW

**Review by ben-alkov — APPROVED** [review]

File: docs/admins.rst [hunk 1]
```diff
@@ -594,3 +594,42 @@ container logs. For instance::
 In this example, this build is stuck talking to the docker client (``docker/client.py``).
 
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
+
+Remote hosts
+------------
+
+Remote hosts are virtual machine hosts that are used by OSBS to build binary images.
```
Comment (rcerven) [inline docs/admins.rst:601]: well they don't have to be virtual

File: docs/admins.rst [hunk 2]
```diff
@@ -594,3 +594,42 @@ container logs. For instance::
...
+During the binary container build phase of OSBS pipeline, podman build command is
```
Comment (rcerven) [inline docs/admins.rst:602]: podman-remote

File: docs/admins.rst (hunk 1 above)
Comment (mkosiarc) [reply docs/admins.rst]: I know, later in the docs I mentioned multiple choices for the hosts. Removed this reference to VMs though.

File: docs/admins.rst (hunk 2 above)
Comment (mkosiarc) [reply docs/admins.rst]: updated.

**Review by rcerven — APPROVED** [review]

**Pull request merged**